storybook/stories/Navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { BrowserRouter } from "react-router-dom"
+import { action } from "@storybook/addon-actions"
+import { Navbar, Environment } from "@misk/core"
+
+export const Production = () => (
+  <BrowserRouter>
+    <Navbar
+      environment={Environment.PRODUCTION}
+      homeName={"Dino Service"}
+      homeUrl={"https://github.com/cashapp"}
+      links={[
+        {
+          name: "First Tab",
+          slug: "first-tab",
+          url_path_prefix: "/first-tab/",
+          category: "Demo"
+        },
+        {
+          name: "Second Tab",
+          slug: "second-tab",
+          url_path_prefix: "/second-tab/",
+          category: "Demo"
+        }
+      ]}
+      navbar_items={[
+        <a onClick={action('click-first-item')}>First Item</a>, 
+        "Second Item"
+      ]}
+      status={"Status Text"}
+    />
+  </BrowserRouter>
+)
+
+export const Staging = () => (
+  <BrowserRouter>
+    <Navbar
```

Comment:
Have you tested with this setup that changes you make to the Navbar in `packages/@misk/core/` appear in the Storybook once rebuilt? That would be the key part to allow use of the Storybook when hacking on local library changes.

Reply:
Good point, will try it out.

Reply:
It definitely works ✨  

It watches the lib folders and hot-reloads the components.

Reply:
So cool! Let’s merge!

Reply:
I have no merge access :(

I'm also not very familiar with whether we should squash this PR into one commit or merge as is.

Reply:
Github squash feature works great! I'll merge 😊